src/components/game/components/board/components/deck/Deck.js

```diff
@@ -1,8 +1,19 @@
+import { phase } from "models/game/selectors";
 import React from "react";
+const phaseSelector = (phase) => {
+  switch (phase) {
+    case 0:
+      return "Start Game";
+    case 2:
+      return "Replace cards";
+    default:
+      return "Play";
+  }
+};
```

Comment:
Clarity:
You could name this function phaseDescription or getPhaseDescription and name the variable phaseId instead of just phase.

It's a suggestion

Reply:
Done